src/main/java/teammates/storage/sqlapi/UsersDb.java

```diff
@@ -140,6 +141,25 @@ public Student getStudentByGoogleId(String courseId, String googleId) {
         return HibernateUtil.createQuery(cr).getResultStream().findFirst().orElse(null);
     }
 
+    /**
+     * Gets a list of students by {@code teamName} and {@code courseId}.
+     */
+    public List<Student> getStudentsByTeamName(String teamName, String courseId) {
+        CriteriaBuilder cb = HibernateUtil.getCriteriaBuilder();
+        CriteriaQuery<Student> cr = cb.createQuery(Student.class);
+        Root<Student> studentRoot = cr.from(Student.class);
+
+        studentRoot.alias("student");
```

Comment:
Why is this needed?

Reply:
Team and User have duplicated attribute `name`. Curious from the previous comment on commenting, should I have commented for this?